hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCreate.java

```diff
@@ -104,4 +107,39 @@ public void testCreateNonRecursive2() throws Exception {
         .close();
     assertIsFile(fs, testFile);
   }
+
+  /**
+   * Attempts to use to the ABFS stream after it is closed.
+   */
+  @Test
+  public void testWriteAfterClose() throws Throwable {
+    final AzureBlobFileSystem fs = getFileSystem();
+    Path testPath = new Path(TEST_FOLDER_PATH, TEST_CHILD_FILE);
+    FSDataOutputStream out = fs.create(testPath);
+    out.close();
+    intercept(IOException.class, () -> out.write('a'));
+    intercept(IOException.class, () -> out.write(new byte[]{'a'}));
+    // hsync is not ignored on a closed stream
+    // out.hsync();
```

Comment:
Why is hsync special?

Reply:
no idea, it just failed for me *and I wasn't worried about it*. flush() is the one I care about as that I know can get called in some places when closed (compression libs etc, so nothing we can fix).

Reply:
Ok, certainly not critical so I'll resolve this comment.